pipelines/cluster-role-binding.yaml

```diff
@@ -0,0 +1,12 @@
+piVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: appsody-admin
+subjects:
+- kind: ServiceAccount
+  name: appsody-sa
```

Comment:
I think we are forking the appsody sample files at this point, we should use "kabanero" instead of "appsody".

Reply:
fixed a few more appsody references outside of the appsody build task itself